src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -59,13 +59,14 @@ extern "C" {
 
 // Used to send the Indication Confirmation
 uint8_t dev_address[RSI_DEV_ADDR_LEN];
-uint16_t ble_measurement_hndl;
+uint16_t rsi_ble_measurement_hndl;
+uint16_t rsi_ble_gatt_server_client_config_hndl;
 
 osSemaphoreId_t sl_rs_ble_init_sem;
 osTimerId_t sbleAdvTimeoutTimer;
 
 static osThreadId_t sBleThread;
-constexpr uint32_t kBleTaskSize = 2048;
+constexpr uint32_t kBleTaskSize = 2560;
```

Comment:
Why the stack size increase?

Reply:
Stack size is increased due to the change in the struct from pointer to variable, the size of the struct is 320b

Reply:
I'm not sure i understand the reasoning. Can you provide a bit more information?

Reply:
The global struct was having a pointer and the memory was not dynamically allocated. Now since it is changed to a variable the eventData takes around 320b causing this increase